deploy/gk-deploy

```diff
@@ -67,6 +69,12 @@ Options:
               The namespace to use for creating resources.
               Defaults is '${NAMESPACE}'.
 
+  --admin-key ADMIN_KEY
+              Secret for heketi administrator user. 
+
```

Comment:
Expand the text to describe what the key does and what it should look like (e.g. is it a 32-byte hex string).